clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
```

Comment:
I don't remember but I suppose that pycharm told me if I want to install this library.